src/Components/ArticleComments/ArticleComments.jsx

```diff
@@ -37,6 +54,14 @@ function ArticleComments({
                   Posted on {moment(`${created_at}`).format("Do MMMM YYYY")}{" "}
                 </p>
                 <p>Votes: {votes}</p>
+                {!isDeleting && (
```

Comment:
this condition will only render the button if the isDeleting is false - so once clicked the button may disappear, rather than be disabled from clicking

Reply:
That makes sense, thanks for flagging. I've taken that out.